forc-plugins/forc-doc/src/tests/expects/impl_trait/mod.rs

```diff
@@ -43,124 +43,125 @@ fn test_impl_traits_default() {
         doc_dir_name,
         project_name,
         vec![
```

Comment:
It won't help for this PR, but for later PRs, should these be sorted so that diffs can be easier on the eyes?